lib/Elevate/Components/DatabaseUpgrade.pm

```diff
@@ -183,7 +183,7 @@ sub _activate_localhost_profile {
 
     if ( $stdout !~ m{MySQL profile activation done} ) {
         die <<~"EOS";
-        Unable to activate a MySQL DB profile for "localhost":
+        Unable to activate a MySQL profile for "localhost":
```

Comment:
Don't we want to yank "MySQL" here?

> Unable to activate a database profile for "localhost":

Reply:
Prior to 120, the feature name is "[Manage MySQL® Profiles](url)". Where we discuss profiles, I elected to keep "MySQL" to keep it consistent with the product and documentation in use on 110.